src/sidebar/components/group-list-section.js

```diff
@@ -1,23 +1,69 @@
 'use strict';
 
-// @ngInject
-function GroupListSectionController() {
-  this.isSelectable = function(groupId) {
-    const group = this.sectionGroups.find(g => g.id === groupId);
-    return !this.disableOosGroupSelection || group.isScopedToUri;
-  };
+const { Fragment, createElement } = require('preact');
+const propTypes = require('prop-types');
+
+const GroupListItem = require('./group-list-item');
+const GroupListItemOutOfScope = require('./group-list-item-out-of-scope');
+
+/**
+ * A labeled section of the groups list.
+ */
+function GroupListSection({
+  analytics,
+  disableOosGroupSelection,
+  heading,
+  sectionGroups,
+  store,
+}) {
+  const isSelectable = groupId =>
+    !disableOosGroupSelection ||
+    sectionGroups.find(g => g.id === groupId).isScopedToUri;
+
+  return (
+    <Fragment>
+      <h2 className="group-list-section__heading">{heading}</h2>
+      <ul className="group-list-section__content">
+        {sectionGroups.map(group => (
+          <li
+            className="dropdown-menu__row dropdown-menu__row--no-border dropdown-menu__row--unpadded"
+            key={group.id}
+          >
+            {isSelectable(group.id) ? (
```

Comment:
The ternary operator here—is it React convention to do so for multi-line options? I found it a little hard to read and was surprised to see a colon on line 39 initially. Not majorly derailing, and I have a hunch this may be the React way...